x-pack/platform/packages/shared/kbn-ai-assistant/src/chat/welcome_message_knowledge_base.tsx

```diff
@@ -20,35 +20,113 @@ import {
 import usePrevious from 'react-use/lib/usePrevious';
 
 import { WelcomeMessageKnowledgeBaseSetupErrorPanel } from './welcome_message_knowledge_base_setup_error_panel';
-import type { UseKnowledgeBaseResult } from '../hooks/use_knowledge_base';
+import { useKibana } from '../hooks/use_kibana';
+import { UseKnowledgeBaseResult } from '../hooks';
 
 export function WelcomeMessageKnowledgeBase({
   knowledgeBase,
 }: {
   knowledgeBase: UseKnowledgeBaseResult;
 }) {
-  const prevIsInstalling = usePrevious(knowledgeBase.isInstalling);
+  const { notifications, ml } = useKibana().services;
+
+  const isInstalling = !!knowledgeBase.status.value?.endpoint && !knowledgeBase.status.value?.ready;
```

Comment:
Wouldn't this mean if, for eg, `deployment_stats?.allocation_status?.allocation_count` became 0, we'd be showing instaling message again?  That's what i'm seeing. 

<img width="664" alt="Screenshot 2025-03-18 at 16 33 19" src="https://github.com/user-attachments/assets/c0434af1-816b-4397-8b19-960eee935fa4" />


`isInstalling` used to be part of the install process: https://github.com/elastic/kibana/blob/main/x-pack/platform/packages/shared/kbn-ai-assistant/src/hooks/use_knowledge_base.tsx#L41

Reply:
We discussed this in our meeting and this is intentional.  The idea is to keep the user in the "installing" process with an "inspect" if the node is being created after being scaled down.  I do find this UX to be confusing as the user already "installed the knowledge base" at this point and we're showing the same message as when they first install it plus an inspect link.  Fine with it as an improvement from the current functionality as showing the installation button.  CC @isaclfreire

Reply:
Could we change the message to "Knowledge base has been installed. Waiting for the model to be deployed on an available ML node." (or something along those lines)

Reply:
That way we clearly indicate what's the holdup, thereby also helping them diagnosing the problem (no available ML nodes)